static/templates/authentication/register.html

```diff
@@ -74,8 +74,9 @@
         you agree to provide Daemo accurate and complete information about you and your business, and you authorize Daemo to share it and transaction
         information related to your use of the payment processing services provided by Stripe. Daemo does not store
         any banking or credit card information.</div-->
-    <div class="authentication-error" layout="row" ng-show="register.error && form.$pristine"
-         ng-bind="register.error"></div>
+    <div class="authentication-error" layout="row" ng-show="register.error=='We are currently in closed beta. If you\'d like an account, email daemo@cs.stanford.edu with a short description of what you\'d like to use Daemo for.' && form.$pristine">
```

Comment:
and what if it's something else? :P 
i'd recommend doing this in the controller